blocks/page-tabs/page-tabs.js

```diff
@@ -15,32 +17,58 @@ function openTab(target) {
   }
 }
 
-function createTabList(sections, active) {
-  const ul = document.createElement('ul');
-  sections.forEach((section) => {
-    const li = document.createElement('li');
-    const a = document.createElement('a');
-    a.href = `#${section.getAttribute('data-name')}`;
-    a.id = section.getAttribute('data-name');
-    a.textContent = section.title;
-    a.addEventListener('click', (e) => {
+function createTabList(block, namedSections, active) {
+  let list = block.querySelector('ul');
+  if (!list) {
+    block.append(list = ul());
```

Comment:
This statement feels a bit off, why not just do it in two lines for clarity?
```suggestion
    list = ul();
    block.append(list);
```